src/components/Error/Error.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './Error.scss'
+
+class Error extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            errorMeg: this.props.errorMeg
+        };
+      }
+      
+      render() {
+        return <span>
+        <svg src="../../assets/images/svg/Exclamation_Circle_Red.svg" />
```

Comment:
This image is still not working for me.  Please import it as a variable at the top of the file and use the variable for the `src` attribute, so Babel can ensure it will work.